docs/fordevs/spras.rst

```diff
@@ -0,0 +1,82 @@
+SPRAS Python Modules
+====================
+
+.. automodule:: spras
```

Comment:
We'll have to add these for each new module manually?

Reply:
I think I found a cleaner way to do this using the `sphinx-apidoc` tool. See the "Auto generate all API module rst files" commit.